src/ripe_rainbow/domain/logic/retail.py

```diff
@@ -25,6 +25,26 @@ def login_and_redirect(self, username, password):
 
         self.waits.redirected_to(self.base_url)
 
+    def select_size(self, size, scale):

```

Comment:
please document the method with something like:

Opens the size window selects the proper scale, clicks the size button and closes the size window.